pkg/resource/notebook_instance_lifecycle_config/custom_delta.go

```diff
@@ -0,0 +1,65 @@
+package notebook_instance_lifecycle_config
+
+import (
+	ackcompare "github.com/aws-controllers-k8s/runtime/pkg/compare"
+)
+
+func modifyDeltaCreate(
+	delta *ackcompare.Delta,
+	a *resource,
+	b *resource,
+) *ackcompare.Delta {
+	if ackcompare.HasNilDifference(a.ko.Spec.OnCreate, b.ko.Spec.OnCreate) {
+		delta.Add("Spec.OnCreate", a.ko.Spec.OnCreate, b.ko.Spec.OnCreate)
+		return delta
+	}
+	var lista []*string
+	var listb []*string
+
+	onCreateLenA := len(a.ko.Spec.OnCreate)
+	onCreateLenB := len(b.ko.Spec.OnCreate)
+	for i := 0; i < onCreateLenA; i++ {
```

Comment:
can be cleaner using [for each loop](https://yourbasic.org/golang/for-loop-range-array-slice-map-channel/#basic-for-each-loop-slice-or-array)

checkout #77 customDescribeSetOutput